tests/limits_test.go

```diff
@@ -13,62 +15,66 @@ var _ = Describe("Limits", func() {
 	Context("with a deployed app", func() {
 
 		var testApp App
-		once := &sync.Once{}
+		var testData TestData
 
 		BeforeEach(func() {
-			// Set up the Limits test app only once and assume the suite will clean up.
-			once.Do(func() {
-				os.Chdir("example-go")
-				appName := getRandAppName()
-				createApp(appName)
-				testApp = deployApp(appName)
-			})
```

Comment:
yes we dont need this. We want each test (It block) to receive its own test data